src/components/TokenBalance.test.tsx

```diff
@@ -737,6 +419,262 @@ describe('FiatExchangeTokenBalance and HomeTokenBalance', () => {
   })
 })
 
+describe('FiatExchangeTokenBalance and HomeTokenBalance', () => {
```

Comment:
nit: can we just do a describe.each and drop the it.each everywhere else?

Reply:
Fixed in 0459fa9b179ffa6e5c9b340679a9329bde61ca79!